docs/source/guide/pec-1-intro.md

```diff
@@ -176,3 +181,34 @@ print(f"Error with PEC:    {abs(ideal_value - pec_value) :.5f}")
 ```
 
 As printed above, PEC reduced the error compared to the unmitigated case.
+
+## Step by step application of PEC
+
+This section demonstrates the use of the {func}`.generate_sampled_circuits` for those who want to generate and see a list of sampled circuits based on the given quasi-probability representaions.
+
+### Generate Sample Circuits
```

Comment:
One of the benefits of the two step approach is that it allows the user to see how many circuits will be run. Let's add a note of that here, and add a line showing `len(sampled_circuits)`.